bin/run_celery_worker_log_parser

```diff
@@ -16,8 +16,8 @@ fi
 
 exec newrelic-admin run-program celery -A treeherder worker \
     -Q \
-parse_job_logs,\
-parse_job_logs_fail,\
+parse_job_log,\
+parse_job_log_fail,\
```

Comment:
Ok, I've reverted back to the previous patch. Let me know if anything else needs adjusting.